winterfell/controllers/authController.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('lodash');
 var passport = require('passport');
+var requestify = require('requestify');
```

Comment:
Mentioned this elsewhere:
I used requestify for now because I was getting access-control-allow-origin problems from using the jquery post methods (ajax post do not work when the domain of your current site does not match the domain of your request). Requestify at least helps with all that complications - we should look for a workaround soon 👍